static/app/views/onboarding/components/useOnboardingQueryParams.tsx

```diff
@@ -1,11 +1,16 @@
 import {useCallback} from 'react';
 
-import {decodeBoolean, decodeList} from 'sentry/utils/queryString';
+import {decodeBoolean, decodeList, decodeScalar} from 'sentry/utils/queryString';
 import useLocationQuery from 'sentry/utils/url/useLocationQuery';
 import {useLocation} from 'sentry/utils/useLocation';
 import {useNavigate} from 'sentry/utils/useNavigate';
 
 type QueryValues = {
+  /**
+   * Used to show the installation mode for certain platforms, e.g. manual or auto.
+   * This is defined inside of a platform file
+   */
+  installationMode: string;
```

Comment:
more refactoring is needed, I would like to get this from the hook `useLoadGettingStarted` but added this here for now.

Reply:
Good idea, but I would not put it there yet as only 2 platforms are using it. let's see first if more similar use cases arise.

Reply:
if we do not define it somewhere,  it won't have types when checking for it in the `setupDocs.tsx`

Reply:
I understand...was trying to avoid the type check in `setupDocs` to be `any`. https://github.com/getsentry/sentry/blob/d65139251f6e157810ab1bdb2af1cef0d7c39544/static/app/views/onboarding/setupDocs.tsx#L41